main.js

```diff
@@ -16,10 +16,10 @@ let perfStats = {
 
 Object.defineProperty(exports, "__esModule", { value: true });
 
-const extension = require('./out/src/extension');
```

Comment:
Revert changes to this file. Related to https://github.com/microsoft/vscode-azurefunctions/pull/1973#issue-387997953